core/src/main/java/hudson/model/StreamBuildListener.java

```diff
@@ -48,7 +48,7 @@ public StreamBuildListener(File out, Charset charset) throws IOException {
     }
 
     /**
-     * @deprecated as of TODO
+     * @deprecated as of 2.328
```

Comment:
The deprecation was added in 2.329 as far as I can tell, commit 3b29e9d460381f83eba9e939f6053b8336d946f4
```suggestion
     * @deprecated as of 2.329
```

Reply:
Thank you for the correction! I will commit this suggestion. The version should indeed be 2.329.